docs/integrations/sources/primetric.md

```diff
@@ -50,12 +50,11 @@ your employees to the right projects.
 
 ### Requirements
 
-* Primetric [client secret](https://app.primetric.com/administrator/integrations)
-* Primetric client id
+- Primetric [client secret](https://app.primetric.com/administrator/integrations)
+- Primetric client id
 
 ## CHANGELOG
 
-| Version | Date       | Pull Request | Subject |
-| :--- |:-----------| :--- | :--- |
-| 0.1.0 | 2022-09-05 | [\#15880](https://github.com/airbytehq/airbyte/pull/15880) | Initial implementation |
-
+| Version | Date       | Pull Request                                               | Subject                |
+| :------ | :--------- | :--------------------------------------------------------- | :--------------------- |
+| 0.1.0   | 2022-09-05 | [\#15880](https://github.com/airbytehq/airbyte/pull/15880) | Initial implementation |
```

Comment:
```suggestion
| 0.1.0   | 2022-09-05 | [15880](https://github.com/airbytehq/airbyte/pull/15880) | Initial implementation |
```
nit: we are inconsistent over different changelogs whether we use # in front of pull request number. it's a larger thing than just here